package.json

```diff
@@ -30,17 +30,12 @@
     "headless.js"
   ],
   "scripts": {
-    "start-webpack": "(cd examples/main && yarn && npm run start-local)",
-    "start": "npm run build-watch && budo dist/bundle.js:dist/lumagl.js --live --open --port 3000 --watch-glob '**/*.{html,css,js,glsl}' ",
-    "build-watch": "babel src -d dist --source-maps inline --copy-files --watch & babel examples -d dist --source-maps inline --copy-files --watch &",
+    "start": "cat < docs/get-started/examples.md",
```

Comment:
Ah, I forgot there's no "layer browser" equivalent in luma.
And actually viewing examples.md, it makes even more sense. 👌